scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtRunner.scala

```diff
@@ -87,10 +87,13 @@ object ScalafmtRunner {
     val scala213 = Scala213
       .withAllowTrailingCommas(true)
     val default = scala213
-      .withAllowOrTypes(true) // New feature in Dotty
-      .withAllowAndTypes(true) // New feature in Dotty
-      .withAllowTraitParameters(true) // New feature in Dotty
-      .withAllowColonForExtractorVarargs(true) // New feature in Dotty
+      // Supported Dotty features.
```

Comment:
unrelated question: for the other changes that @MichalBednarz and @tgodzik have been working on (like `using` and `enum`), do we need to add these properties?

I can see one for `.withAllowEnums`, and something about `.withAllowGivenUsing` (although i am not sure if that's the same as `using`).

Reply:
I think we will use the separate Dotty dialect for now (until parser is fully stable) and require users to add `runner.dialect`. I think @olafurpg already included it in another PR for macros.